docs/sources/features/datasources/elasticsearch.md

```diff
@@ -203,6 +203,8 @@ Querying and displaying log data from Elasticsearch is available via [Explore]({
 
 Select the Elasticsearch data source, change to Logs using the Metrics/Logs switcher, and then optionally enter a lucene query into the query field to filter the log messages.
```

Comment:
I've imagined that we should rewrite this sentence and replace the images that show the switcher but maybe @oddlittlebird have some input about this?

Reply:
In that case, maybe we use the same format as in the [Loki](https://grafana.com/docs/grafana/latest/features/datasources/loki/#querying-logs) section (image removed)?

Reply:
yeah that sounds good to me. Thank you for helping with this.